docs/manual/form-generator/forms.de.md

```diff
@@ -67,6 +67,7 @@ immer als Attachment angehängt.
 | Rohdaten                 | Die E-Mail enthält die unbearbeiteten Daten, das heißt, die Inhalte der einzelnen Formularfelder werden einfach untereinander aufgelistet. |
 | XML-Datei                | Der E-Mail ist eine XML-Datei mit den Formulardaten angehängt. |
 | CSV-Datei                | Der E-Mail ist eine CSV-Datei mit den Formulardaten angehängt. |
+| CSV-Datei (Microsoft Excel) | Der E-Mail ist eine CSV-Datei im Microsoft-Excel-Format mit den Formulardaten angehängt. |
```

Comment:
This is 4.10 only. Should I write this directly into the text or is there a snippet for that?

Reply:
There's a snippet :) `{{< version "4.10" >}}`

Reply:
Added in d459a3836123587361269c4205532259e3f14a19